CreeDictionary/CreeDictionary/migrations/0003_add_wordform_paradigms.py

```diff
@@ -1,6 +1,6 @@
 from django.db import migrations
 
-from CreeDictionary.ensure_data import (
+from ..ensure_data import (
```

Comment:
This one only has two dots but the others have three 🤔

Reply:
The other relative imports are in `management/commands/` whereas this is in `migrations`. Each dot corresponds to a directory out. This is just one directory out.